ruleset/sca/amazon/cis_amazon_linux_2022.yml

```diff
@@ -0,0 +1,5084 @@
+            "AC.L2-3.1.3",
+            "MP.L2-3.8.2",
+          ]
+      - hipaa: ["164.308(a)(3)(i)", "164.308(a)(3)(ii)(A)", "164.312(a)(1)"]
+      - pci_dss_3.2.1: ["7.1", "7.1.1", "7.1.2", "7.1.3"]
+      - pci_dss_4.0: ["1.3.1", "7.1"]
+      - soc_2: ["CC5.2", "CC6.1"]
+    references:
+      - See the fstab(5) and edquota(8) manual pages for more information.
+    condition: all
+    rules:
+      - "c:findmnt --kernel /home -> r:usrquota"
+
+  # 1.1.7.5 Ensure grpquota option set on /home partition.
+  - id: 31027
+    title: "Ensure grpquota option set on /home partition."
+    description: "The grpquota mount option allows for the filesystem to have disk quotas configured."
+    rationale: "To ensure the availability of disk space on /home, it is important to limit the impact a single user or group can cause for other users (or the wider system) by accidentally filling up the partition. Quotas can also be applied to inodes for filesystems where inode exhaustion is a concern."
+    remediation: "Edit the /etc/fstab file and add grpquota to the fourth field (mounting options) for the /home partition: # <device> /home <fstype> defaults,rw,grpquota,grpquota,nodev,relatime 0 0 . Run the following command to remount /home with the configured options: # mount -o remount /home . Create the quota database with # quotacheck -cugv /home , then, restore SELinux context on the quota database file with # restorecon /home/aquota.group and finally enable quotas on the partition with # quotaon -vug /home"
+    compliance:
+      - cis: ["1.1.7.5"]
+      - cis_csc_v8: ["3.3"]
+      - cis_csc_v7: ["14.6"]
+      - nist_sp_800-53: ["AC-3", "MP-2"]
+      - cmmc_v2.0:
+          [
+            "AC.L1-3.1.1",
+            "AC.L1-3.1.2",
+            "AC.L2-3.1.5",
+            "AC.L2-3.1.3",
+            "MP.L2-3.8.2",
+          ]
+      - hipaa: ["164.308(a)(3)(i)", "164.308(a)(3)(ii)(A)", "164.312(a)(1)"]
+      - pci_dss_3.2.1: ["7.1", "7.1.1", "7.1.2", "7.1.3"]
+      - pci_dss_4.0: ["1.3.1", "7.1"]
+      - soc_2: ["CC5.2", "CC6.1"]
+    references:
+      - See the fstab(5) and edquota(8) manual pages for more information.
+    condition: all
+    rules:
+      - "c:findmnt --kernel /home -> r:grpquota"
+
+  # 1.1.8.1 Ensure nodev option set on /dev/shm partition.
+  - id: 31028
+    title: "Ensure nodev option set on /dev/shm partition."
+    description: "The nodev mount option specifies that the filesystem cannot contain special devices partitions."
+    rationale: "Since the /dev/shm filesystem is not intended to support devices, set this option to ensure that users cannot attempt to create block or character special devices in /dev/shm."
+    remediation: "Edit the /etc/fstab file and add nodev to the fourth field (mounting options) for the /dev/shm partition: # <device> /dev/shm <fstype> defaults,rw,nosuid,nodev,noexec,relatime 0 0 . Run the following command to remount /dev/shm with the configured options: # mount -o remount /dev/shm"
```

Comment:
There are extra spaces before the end of the sentence dots.

Reply:
Solved